examples/pubsub-simple/README.md

```diff
@@ -75,14 +71,12 @@ dapr run --app-id python-publisher --app-protocol grpc --dapr-grpc-port=3500 pyt
 
 <!-- STEP
 expected_stdout_lines: 
-  - '✅  app stopped successfully: python-publisher'
```

Comment:
Why the publisher is not stopped anymore?

Reply:
Because we are publishing only a finite number of events and it exits itself. Previously we were publishing an infinite number.